ext/spl/php_spl.stub.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+function iterator_to_array(Traversable $iterator, bool $use_keys = true): array {}
+
+function iterator_count(Traversable $iterator): int {}
+
+function iterator_apply(Traversable $iterator, callable $function, ?array $args = []): int {}
+
+/**
+ * @param object|string $instance
+ * @return array|false
+ */
+function class_parents($instance, bool $autoload = true) {}
+
+/**
+ * @param object|string $what
+ * @return array|false
+ */
+function class_implements($what, bool $autoload = true) {}
+
+/**
+ * @param object|string $what
+ * @return array|false
+ */
+function class_uses($what, bool $autoload = true) {}
+
+function spl_classes(): array {}
+
+/** @return array|false */
+function spl_autoload_functions() {}
+
+function spl_autoload(string $class_name, string $file_extensions): void {}
+
+function spl_autoload_extensions(string $file_extensions): string {}
+
+function spl_autoload_call(string $class_name): void {}
+
+function spl_autoload_register(callable $autoload_function = UNKNOWN, bool $throw = true, bool $prepend = false): bool {}
```

Comment:
Is `$autoload_function` actually constrained to `callable`?

Reply:
`zend_parse_parameters_ex` just specifies a zval, but the implementation looks pretty strict about it being a callable, should I remove the type declaration and add a `@param` for it instead?

Reply:
I'm not 100% sure, but I think `callable` types must be parsed as such (i.e. using an "f" type specifier or `Z_PARAM_FUNC`).

Reply:
Ok I've dropped the type now, added a `@param` to clarify the expected value